src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -2499,24 +2499,13 @@ include::./references/CatalogReference.adoc[]
 === Scheduler Portal Properties
 include::./references/SchedulerPortalReference.adoc[]
 
-=== Node Sources
-
-The ProActive Resource Manager supports ProActive Nodes aggregation from heterogeneous environments.
-As a node is just a process running somewhere, the process of communication to such nodes is unified.
-The only part which has to be defined is the procedure of nodes deployment which could be quite
-different depending on infrastructures and their limitations. After installation of the server and node parts it is
-possible to configure an automatic nodes deployment. Basically, you can say to the Resource Manager how to
-launch nodes and when.
-
-In the ProActive Resource Manager, there is always a default node source consisted of DefaultInfrastructureManager
-and Static policy. It is not able to deploy nodes anywhere but makes it possible to add existing nodes to the RM.
-
```

Comment:
I think this paragraph can be revised and moved into section 4 (Deploy ProActive Nodes) introduction.
For example, before the paragraph "A node source is defined by two parts of configurations..."